styles/main.scss

```diff
@@ -1,9 +1,44 @@
-$color: blue;
+#rajoyirize-container {
+    font-family: Arial;
+    font-size: 16px;
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    overflow: hidden;
+    
+    &.hidden {
```

Comment:
The idea of `hidden` was to be reused by the other elements. However, as we have a container wrapping everything, maybe this is fine.